dictionary/Dictionary.js

```diff
@@ -45,6 +45,24 @@ export default class Dictionary {
 	 *         evaluated using the provided parameters.
 	 */
 	get(key, parameters = {}) {}
+
+	/**
+	 * Tests whether the specified localization phrase exists in dictionary
+	 *
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {boolean} {@code true} if the the key (phrase) exists.
+	 */
+	has(key) {}
+
+	/**
+	 * Get directory scope for specific key
+	 *
+	 * @private
+	 * @param {string} key The key identifying the localization phrase.
+	 * @return {function(*)|boolean} {@code false} if key not exists,
+	 *         otherwise return {@code function} scope
+	 */
+	_getScope(key) {}
```

Comment:
An interface is no place for a private method. Please remove this method from this file (and move the relevant documentation to the file containing the implementation).